processor/src/test/java/org/mapstruct/ap/test/value/EnumToEnumMappingTest.java

```diff
@@ -0,0 +1,264 @@
+/**
+ *  Copyright 2012-2016 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.test.value;
+
+import static org.fest.assertions.Assertions.assertThat;
+
+import javax.tools.Diagnostic.Kind;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mapstruct.ap.testutil.IssueKey;
+import org.mapstruct.ap.testutil.WithClasses;
+import org.mapstruct.ap.testutil.compilation.annotation.CompilationResult;
+import org.mapstruct.ap.testutil.compilation.annotation.Diagnostic;
+import org.mapstruct.ap.testutil.compilation.annotation.ExpectedCompilationOutcome;
+import org.mapstruct.ap.testutil.runner.AnnotationProcessorTestRunner;
+
+/**
+ * Test for the generation and invocation of enum mapping methods.
+ *
+ * @author Gunnar Morling
+ */
+@IssueKey("128")
+@WithClasses({ OrderEntity.class, OrderType.class, OrderDto.class, ExternalOrderType.class })
+@RunWith(AnnotationProcessorTestRunner.class)
+public class EnumToEnumMappingTest {
+
+    @Test
+    @WithClasses( OrderMapper.class )
```

Comment:
I changed it because its a bit harder to debug (figuring out which mapper fails). This is why I comment out the Eclipse version as well when I'm designing the UT. However, I can put them back now..